deploy/template.yaml

```diff
@@ -901,80 +901,6 @@ Resources:
             Period: 60
             Stat: Sum
 
-  # Metrics for frontend-vital-signs
-  EventLoopDelayMetricFilter:
```

Comment:
What about the other stuff with the PR, i wonder if a revert of this PR would just work https://github.com/govuk-one-login/ipv-cri-address-front/pull/940

Reply:
We are not reverting the changes, just removing the MetricFilter and that's it.